tests/ui/UnreadIndicatorsTest.js

```diff
@@ -19,6 +19,8 @@ import * as Report from '../../src/libs/actions/Report';
 import * as CollectionUtils from '../../src/libs/CollectionUtils';
 import DateUtils from '../../src/libs/DateUtils';
 
+// We need a large timeout here as we are lazy loading React Navigation screens and this test is running against the entire mounted App
+jest.setTimeout(30000);
```

Comment:
I tried leaving this in `beforeAll` and moving it to `beforeEach`, and in both scenarios I saw some tests timing out after 5 seconds. After moving it to the top-level of the module, it works as expected. This matches what you'd expect from [the documentation](https://jestjs.io/docs/jest-object#jestsettimeouttimeout):

> Set the default timeout interval (in milliseconds) for all tests and before/after hooks in the test file. This only affects the test file from which this function is called